requirements/binary.txt

```diff
@@ -1 +1,22 @@
-mysql-connector-python==8.0.16 --hash=sha256:ea7d80e2b406dba3b20836e95d76e926d0f80034e5a015c0d8309d820225cb5b
\ No newline at end of file
+# MySQL driver written in Python
+# Code: https://github.com/mysql/mysql-connector-python
+mysql-connector-python==8.0.16 \
```

Comment:
I'm not sure why this isn't in ``all.txt``. It is possible that 8.x no longer requires the MySQL C libraries.

Reply:
IIRC the split into different requirement files was based on doing things in different docker layers, so expensive an rarely changing libraries with C dependencies could be cached longer in local dev environments. But this was before binary wheels became a real thing. So probably this is a thing of the past, which was never updated.